src/Request.php

```diff
@@ -3,7 +3,7 @@
 namespace React\HttpClient;
 
 use Evenement\EventEmitterTrait;
-use Guzzle\Parser\Message\MessageParser;
+use GuzzleHttp\Psr7 as g7;
```

Comment:
I'm sorry to nitpick but can we come up with another alias? I used `gPsr` other places. At the rate Guzzle releases major versions this could quickly get mixed up with misunderstanding "Guzzle7".